tiatoolbox/wsicore/wsireader.py

```diff
@@ -129,13 +129,13 @@ def _relative_level_scales(self, resolution, units):
                 units.
 
         Examples:
-            >>> from tiatoolbox.wsicore.wsireader import get_wsireader
-            >>> wsi = get_wsireader(input_img="./CMU-1.ndpi")
+            >>> from tiatoolbox.wsicore import wsireader
```

Comment:
this should be replaced by get_wsireader

Reply:
Fixed in this commit https://github.com/TIA-Lab/tiatoolbox/pull/123/commits/3fbfef2da40097681f72adf4f0d16bd7377f42e1